get-started/index.md

```diff
@@ -56,127 +56,156 @@ redirect_from:
 
-![Container stack example](https://www.docker.com/sites/default/files/Container%402x.png)
+For full [Kubernetes integration on Docker for Mac](https://docs.docker.com/docker-for-mac/kubernetes/){:target="_blank"},
+install [17.12.0-ce Edge](https://docs.docker.com/docker-for-mac/release-notes/#edge-release-notes){:target="_blank"} or higher.
 
-Containers can share a single kernel, and the only information that needs to be
-in a container image is the executable and its package dependencies, which never
-need to be installed on the host system. These processes run like native
-processes, and you can manage them individually by running commands like `docker
-ps`&#8212;just like you would run `ps` on Linux to see active processes.
-Finally, because they contain all their dependencies, there is no configuration
-entanglement; a containerized app "runs anywhere."
+[Install Docker](/engine/installation/index.md){: class="button outline-btn"}
+<div style="clear:left"></div>
 
-## Setup
+### Test Docker version
 
-Before we get started, make sure your system has the latest version of Docker
-installed.
+Ensure that you have a supported version of Docker:
 
-[Install Docker](/engine/installation/index.md){: class="button outline-btn"}
-<div style="clear:left"></div>
-> **Note**: version 1.13 or higher is required
+```shell
+$ docker --version
+Docker version 17.12.0-ce, build c97c6d6
+```
+
+Run `docker version`(without `--`) or `docker info` to view even more details
+about your docker installation:
 
-You should be able to run `docker run hello-world` and see a response like this:
-> **Note**: You may need to add your user to the `docker` group in order to call this command without sudo. [Read more](https://docs.docker.com/engine/installation/linux/linux-postinstall/)
+```shell
+$ docker info
+Containers: 0
+ Running: 0
+ Paused: 0
+ Stopped: 0
+Images: 0
+Server Version: 17.12.0-ce
+Storage Driver: overlay2
+...
+```
+
+> **Note**: To avoid permission errors (and the use of sudo), you may need to
+> add your user to the `docker` group. [Read more](https://docs.docker.com/engine/installation/linux/linux-postinstall/){:target="_blank"}.
 
-> **Note**: If there are networking issues in your setup, `docker run hello-world` may fail to execute successfully. In case you are behind a proxy server and you suspect that it blocks the connection, check the [next part](https://docs.docker.com/get-started/part2/) of the tutorial.
+### Test Docker installation
+
+Test that your installation works by running a simple Docker image:
 
 ```shell
 $ docker run hello-world
 
+Unable to find image 'hello-world:latest' locally
+latest: Pulling from library/hello-world
+ca4f61b1923c: Pull complete
+Digest: sha256:ca0eeb6fb05351dfc8759c20733c91def84cb8007aa89a5bf606bc8b315b9fc7
+Status: Downloaded newer image for hello-world:latest
+
 Hello from Docker!
 This message shows that your installation appears to be working correctly.
+...
+```
+
+List the `hello-world` image:
```

Comment:
Is this a good time to talk a tiny bit about image vs container? It may not be obvious what the difference is just with these two commands.

Reply:
I do that above and would hate to repeat myself and clutter the page with redundant information. I am hoping that the instructions disambiguate: run image, list image, list container, etc. Do I persuade?